src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -246,6 +246,24 @@
                 }, that.options.searchTimeOut);
             });
 
+            header.off('mouseup', 'input').on('mouseup', 'input', function (event) {
+                var $input = $(this),
+                oldValue = $input.val();
+
+                if (oldValue == "") return;
```

Comment:
Please use the equal value and equal type validation and the correct format

if (something === thing) {
...
}